code/style.css

```diff
@@ -6,8 +6,7 @@
 
 body {
   font-family: "Montserrat", sans-serif;
-  background: #0026ff;
-  color: white;
+  background-image: url("/pexels.jpg");  color: white;
```

Comment:
the image is not rendering correctly, please take a look at this path. Perhaps you meant `./pexels.jpg`?